src/ChestShop/Main.php

```diff
@@ -106,7 +106,7 @@ public function onEnable() : void{
 			$this->updateConfig($file);
 		}
 
-		$shops = yaml_parse_file($this->getDataFolder().'shops.yml');
+		$shops = @yaml_parse_file($this->getDataFolder().'shops.yml');
```

Comment:
What's the reason for this?

Reply:
@Muqsit https://hastebin.com/qafoketipo.tex

Reply:
https://github.com/Muqsit/ChestShop/blob/master/src/ChestShop/Main.php#L102